src/components/search-form/search-form.js

```diff
@@ -32,104 +31,44 @@ export default class SearchForm extends Component {
     }),
     onSearch: PropTypes.func.isRequired,
     searchString: PropTypes.string,
-    filter: PropTypes.object,
-    searchfield: PropTypes.string,
+    searchField: PropTypes.string,
+    searchFilter: PropTypes.object, // TODO: shape it
```

Comment:
When does this get shaped?

Reply:
HA, great catch on this. I'll push an update doing that.